retrieval_service/datastore/providers/neo4j_graph.py

```diff
@@ -83,6 +85,24 @@ async def create_amenities(tx, amenities):
                     hour=amenity.hour,
                 )
 
+                # Create Category node
+                await tx.run(
+                    """
+                    MERGE (c:Category {name: $category})
+                    """,
+                    category=amenity.category,
+                )
+
+                # Create BELONGS_TO relationship
+                await tx.run(
+                    """
+                    MATCH (a:Amenity {id: $id}), (c:Category {name: $category})
+                    MERGE (a)-[:BELONGS_TO]->(c)
```

Comment:
Should this be MERGE or CREATE?

Reply:
Merge is a better option in case csv files have duplicated data.